includes/otb-utility.php

```diff
@@ -277,6 +277,70 @@ function dc_metadata_to_csv( $dspace_array, $dc_type ) {
 }
 
 /**
+ * @param $number
+ *
+ * @return float|string|void
+ */
+function determineFileSize( $number ) {
```

Comment:
the format that I've been following is to camelCase functions if they are inside a class and under_score functions if they aren't.  If this can replace `\BCcampus\OpenTextBooks\Views\Books->determineFileSize` then all references should be updated to point to this new, renamed function `determine_file_size`.